### DISCUSSION TITLE
Docker in Mocaccino

### DISCUSSION THREAD

**Discussion by VolvoBirkenstock** [mocaccinoOS/mocaccino#98] [General] [answered]
Hello again, Mocaccino Gurus! 

If I wanted to download the docker container and play around inside the container, would I need to get the container package from https://packages.mocaccino.org/luet-official/? Or are more packages required?  

Is the docker container intended only for development of packages for mocaccino?

**Comment by msdobrescu** [comment, marked as answer]
Please read [here](https://www.mocaccino.org/docs/micro/docker/).
Also [here](https://www.mocaccino.org/docs/desktop/building_packages/).
Those are related to MOS.
You can use luet on any other linux and need docker installed by that distro ways or how you chose.
Docker is the mean to build isolated from the host OS. In  our case, we avoid to break the current installation by broken builds or other stuff that may remain from some reason.

**Reply by VolvoBirkenstock** [reply]
Thank you, @msdobrescu
